src/Utils/CommandInputValidator.php

```diff
@@ -83,4 +88,39 @@ public function validateFullName(?string $fullName): string
 
         return $fullName;
     }
+
+    public function validateRole(?string $role): string
+    {
+        if (empty($role)) {
+            throw new InvalidArgumentException('The role can not be empty.');
+        }
+
+        $allowedRoles = ['editor', 'admin'];
```

Comment:
For future-proofing you could set these roles in an environment variable. Or at least put the list into a class variable; it's also used in `interact`.

Reply:
Moved to a class constant.

Originally I tried to get the values through DI and the one of the Security services. Didn't work as I wanted because it's a subset of the defined roles we allow here.

I don't like moving them to `env` because they would still have to match what is defined in `security.yaml`. Also I consider `env` values to be something you can configure. But Symfony's docs encourage you to do stuff like `#[IsGranted('ROLE_ADMIN')]` in code, so having the roles configurable would clash with that.